js/app/src/Blocks.svelte

```diff
@@ -502,7 +502,16 @@
 				});
 		} else {
 			if (dep.backend_fn) {
-				make_prediction();
+				if (dep.trigger_mode === "once") {
+					if (!dep.pending_request) make_prediction();
+				} else if (dep.trigger_mode === "multiple") {
+					do {
+						dep.pending_request = false;
+						make_prediction();
+					} while (dep.pending_request);
+				} else if (dep.trigger_mode === "always_last") {
+					make_prediction();
```

Comment:
I'm a little confused here

It seems like the original behavior (which was to just call `make_prediction()`) mapes to the case of `dep.trigger_mode === "always_last"`.

But that should be the case, it should map to `dep.trigger_mode === "once"`